hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMMetrics.java

```diff
@@ -1505,6 +1509,14 @@ public void incNumRecoverLeaseFails() {
     numRecoverLeaseFails.incr();
   }
 
+  public void incNumSuccessfulIterationsDirDeletingService() {
+    numSuccessfulIterationsDirDeletingService.incr();
```

Comment:
We do not need this metrics, already separate PR is there considering all required metrics.

Reply:
done.